gitprompt.sh

```diff
@@ -354,9 +354,12 @@ function setGitPrompt() {
   OLD_GIT_PROMPT_IGNORE_SUBMODULES="${GIT_PROMPT_IGNORE_SUBMODULES}"
   unset GIT_PROMPT_IGNORE_SUBMODULES
 
+  OLD_GIT_PROMPT_SHOW_TRACKING=${GIT_PROMPT_SHOW_TRACKING}
+  unset GIT_PROMPT_SHOW_TRACKING
+
   if [[ -e "${repo}/.bash-git-rc" ]]; then
     # The config file can only contain variable declarations on the form A_B=0 or G_P=all
-    local CONFIG_SYNTAX="^(FETCH_REMOTE_STATUS|GIT_PROMPT_SHOW_UNTRACKED_FILES|GIT_PROMPT_IGNORE_SUBMODULES|GIT_PROMPT_IGNORE)=[0-9a-z]+$"
+    local CONFIG_SYNTAX="^(FETCH_REMOTE_STATUS|GIT_PROMPT_SHOW_UNTRACKED_FILES|GIT_PROMPT_IGNORE_SUBMODULES|GIT_PROMPT_SHOW_TRACKING|GIT_PROMPT_IGNORE)=[0-9a-z]+$"
```

Comment:
GIT_PROMPT_SHOW_TRACKING is used here but unset un line 358. So it will always be empty, woulnt it?

Reply:
No I think we used this to be able to set some of the variables in a local config file (.bash-git-rc) the config syntax line shows the properties you are allowed to override in that file.

Reply:
:) you are absolutely right. I blindly searched for the term and didn't notice that it is not used as a variable but as a string. thx